### DISCUSSION TITLE
ant-design 6 有计划吗？

### DISCUSSION THREAD

**Discussion by kaoding** [ant-design/ant-design#51590] [General]
antd 3有26个小版本，antd 4有24个小版本，antd 5现在到了22小个版本，都是一年左右升级大版本的，后面有antd 6的计划吗？

**Comment by petercat-assistant** [comment]
目前没有找到关于 Ant Design 6 的具体计划或信息。如果有新的版本计划，通常会在 Ant Design 的官方 GitHub 仓库或官网上公布。您可以关注这些渠道以获取最新的更新。

如果您还有其他关于 Ant Design 的问题，欢迎随时询问。

For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.

**Comment by li-jia-nan** [comment]
v6 比较被动，需要等待 React19 正式版发布

**Reply by totorofly** [reply]
> v6 比较被动，需要等待 React19 正式版发布

尴尬，新项目用了 Next15.js，结果最需要的 Antd 还不支持😮‍💨

**Comment by arvinxx** [comment]
感觉明年会有了吧？

**Comment by zombieJ** [comment]
暂时没有。
目前要先观望一下 React，v5 兼容到 React 16，兼容代码和 React 19 rc 版本无法兼容。上 19 后要舍弃旧版 React。

**Reply by li-jia-nan** [reply]
19 正式版有没有可能把 createRoot API 路径改回去？

**Reply by kaoding** [reply]
antd 本身有没有大的改进计划呢？

**Reply by li-jia-nan** [reply]
不是每个月都在改进吗？

**Reply by arvinxx** [reply]
> antd 本身有没有大的改进计划呢？

这个肯定会有的吧

**Comment by kaoding** [comment]
https://github.com/ant-design/ant-design/discussions/51919

**Comment by Wxh16144** [comment]
🎉 Ant Design 6.0 来了！ 🎉 #55805
